app/core/logging.py

```
@@ -0,0 +1,222 @@
+
+This module provides structured JSON logging configuration with:
+- Environment-specific log levels
+- Sensitive data masking
+- Process information tracking
+- Structured JSON output for production
+- Pretty console output for development
+"""
+
+import logging
+import os
+from typing import Any, Dict, List, Optional
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)